plugins/CoreHome/templates/_menu.twig

```diff
@@ -1,21 +1,21 @@
 {% macro submenuItem(name, url, anchorlink) %}
     {% if name|slice(0,1) != '_' %}
-        <li>
+        <li role="menuitem" title="{{ name|translate|e('html_attr') }}">
             <a class="item" href="{% if anchorlink %}#{% else %}index.php?{% endif %}{{ url|urlRewriteWithParameters|slice(1) }}">
-                {{ name|translate }}
+                {{ name|translate|e('html_attr') }}
```

Comment:
There should be no `|e('html_attr)` here. It should be only used when it will be used within an HTML attribute like above in the title attribute.